### PR TITLE
fix: fix and refactor AoT css

### PR DESCRIPTION
**Pull request by zbettenbuk**

### CODE REVIEW

**Comment by zbettenbuk** [comment]
image

File: css/_toolbars.scss [hunk 1]
```diff
@@ -270,7 +270,6 @@
     }
 }
 
-.always-on-top-toolbox,
 .filmstrip-toolbox {
     background-color: $newToolbarBackgroundColor;
     box-sizing: border-box;
```
Comment (hristoterezov) [inline css/_toolbars.scss]: Shouldn't we remove all `i` style from here?

Comment (hristoterezov) [reply css/_toolbars.scss]: Also I guess the filmstrip only toolbar is now having the same issues as the AOT toolbar. Did you test?

File: css/_toolbars.scss [hunk 2]
```diff
@@ -304,36 +303,6 @@
...
 
-.always-on-top-toolbox {
-    flex-direction: row;
-    left: 50%;
-    position: absolute;
-    top: 10px;
-    transform: translateX(-50%);
-    z-index: $toolbarZ;
-
-    i {
```
Comment (hristoterezov) [inline css/_toolbars.scss]: Are those styles gone? Or they are inherited from somewhere?

**Pull request merged**